eng/helix/content/runtests.cmd

```diff
@@ -4,35 +4,36 @@ setlocal enabledelayedexpansion
 
 REM Use '$' as a variable name prefix to avoid MSBuild variable collisions with these variables
 set $target=%1
-set $sdkVersion=%2
-set $runtimeVersion=%3
-set $queue=%4
-set $arch=%5
-set $quarantined=%6
-set $ef=%7
-set $aspnetruntime=%8
-set $aspnetref=%9
-REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
-shift
+set $runtimeVersion=%2
+set $queue=%3
+set $arch=%4
+set $quarantined=%5
+set $ef=%6
+set $aspnetruntime=%7
+set $aspnetref=%8
 set $helixTimeout=%9
+REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
 
-set DOTNET_HOME=%HELIX_CORRELATION_PAYLOAD%\sdk
-set DOTNET_ROOT=%DOTNET_HOME%\%$arch%
+set DOTNET_ROOT=%HELIX_CORRELATION_PAYLOAD%\dotnet
+set DOTNET_HOME=%DOTNET_ROOT%
```

Comment:
In our previous conversations about this change and about incorrectly changing the payload folders, I got the impression we were getting out from under the payload folders with this PR. Won't we hit leftover files still❔

I'm hoping this is fine and that I misunderstood you and @MattGal 😈

Reply:
If we want to install the runtime to a different directory we can still do that in a different PR, that's orthogonal to using the sdk support